SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -744,12 +730,6 @@ FRPCErrorInfo USpatialSender::SendRPC(const FPendingRPCParams& Params)
 	const FRPCInfo& RPCInfo = ClassInfoManager->GetRPCInfo(TargetObject, Function);
 	bool bUseRPCRingBuffer = GetDefault<USpatialGDKSettings>()->UseRPCRingBuffer();
 
-	if (RPCInfo.Type == ERPCType::CrossServer)
-	{
-		SendCrossServerRPC(TargetObject, Function, Params.Payload, Channel, Params.ObjectRef);
-		return FRPCErrorInfo{ TargetObject, Function, ERPCResult::Success };
-	}
-
 	if (bUseRPCRingBuffer && RPCService != nullptr)
 	{
 		if (SendRingBufferedRPC(TargetObject, Function, Params.Payload, Channel, Params.ObjectRef))
```

Comment:
The cross server rpcs are processed through here now?

Reply:
If so, we might want to change this function's name

Reply:
no, cross server rpcs are not processed at all through the SpatialSender anymore. They are now processed in the CrossServerRPCSender